### DISCUSSION TITLE
What´s the better or a good way to create new images for new GUI menus?

### DISCUSSION THREAD

**Discussion by rafa4mat** [WinEdt-Team/WinEdt-Support#90] [3.1 Lounge]
What´s the better or a good way (you found) to create new images for new GUI menus?.

I ask this for get a good (sharp and legible) result in the menu and not blur and difficult to read.

My first test was the image below (a little blur and not as sharp as I would like):

![snap004388](https://github.com/user-attachments/assets/a9ebc65f-9204-482a-8692-03ecd536d2a4)

Better an .bmp directly from LaTeX?, as standalone INSTEAD of usign design software as photoshop, illustrator,,..?

**Comment by rafa4mat** [comment]
I answer myself.

The way I´m using to obtain
![snap004406](https://github.com/user-attachments/assets/8b6fbd31-f42d-489d-a26d-fba6436d762a)

is compile

	\documentclass[a4paper,12pt]{standalone}

	\usepackage[utf8]{inputenc}
	\usepackage[spanish]{babel}

	\usepackage{mathpazo}

	\usepackage{amsmath}
	\usepackage{xcolor}
	\usepackage[normalem]{ulem}

	\usepackage{array}
	\usepackage{tabularray}


	\begin{document}


	\SetTblrInner{rowsep=0pt}
	\begin{tblr}
	{rows = {abovesep=1pt,belowsep=2pt},colspec=
	{Q[c,wd=1.90cm]Q[c,wd=1.90cm]Q[c,wd=1.90cm]Q[c,wd=1.90cm]}}
	  %\hline
	  % after \\: \hline or \cline{col1-col2} \cline{col3-col4} ...
	  \uline{uline} & \uwave{uwave}  & \sout{sout}  & \dashuline{dashuline} \\
	  \uuline{uuline} &  & \xout{xout}  & \dotuline{dotuline} \\
	  %\hline
	\end{tblr}

	\end{document} 


Then, using [https://imagemagick.org/index.php](https://imagemagick.org/index.php)

    magick convert -density 183 WinEdt-Tools-Ulem01.pdf WinEdt-Tools-Ulem01.png
(because the .bmp option give any problem)

after that, if neccessary, use a design software to adjust any measure.

to, finally obtain the .bmp.


I still can´t control well heigths, as you can see at the below image
![snap004408](https://github.com/user-attachments/assets/62d13401-7576-42ba-9b8e-e97bc9912b4d)

**Reply by Alex-Simonic** [reply]
I used the document GUI.tex (available in Documents -> Samples -> TeX Symbols menu). I compiled it into PDF and previewed it in WinEdt PDF at a suitable magnification (which depends on your screen resolution). Then I made a screenshot and pasted it in MS Paint. Clip (copy and paste) each group rectangle and save it as a 24-bit bitmap. I made some custom Delphi code to optimize the bitmap size by reducing item spacing. The GUI version for WinEdt 11 was recreated at higher resolution (older versions were not high-dpi friendly). It was labor-intensive and hard on my eyes but that's how it was done...
